interface/frontend/src/components/common/OpenDICOM.vue

```diff
@@ -70,27 +70,21 @@
         this.stack.currentImageIdIndex = newIndex
       },
       'view.paths' (val) {
-        const element = this.$refs.DICOM
-        this.pool = Array(this.view.paths.length)
-
-        if (this.view.sliceIndex >= 0) {
-          this.stack.currentImageIdIndex = this.view.sliceIndex
-        } else {
-          this.stack.currentImageIdIndex = this.view.paths.indexOf(this.view.state)
-          if (this.stack.currentImageIdIndex < 0) this.stack.currentImageIdIndex = 0
-        }
-
-        this.stack.imageIds = this.view.paths.map((path) => {
-          return this.view.type + this.view.prefixCS + path
-        }, this)
-        cornerstone.disable(element)
-        this.initCS(element)
+        this.updateViewer()
       }
     },
     computed: {
       async info () {
         if (typeof this.pool[this.stack.currentImageIdIndex] === 'undefined') {
-          const hola = await this.$axios.get(this.view.prefixUrl + this.view.paths[this.stack.currentImageIdIndex])
+          // workaround for now; sometimes we have full url, sometimes filepath
+          // const hola = await this.$axios.get(this.view.prefixUrl + this.view.paths[this.stack.currentImageIdIndex])
+          var hola
+          if (this.view.paths.length > 0 && this.view.paths[this.stack.currentImageIdIndex].indexOf('=') !== -1) {
+            hola = await this.$axios.get(this.view.paths[this.stack.currentImageIdIndex])
+          } else {
+            hola = await this.$axios.get(this.view.prefixUrl + this.view.paths[this.stack.currentImageIdIndex])
```

Comment:
what about the case where `this.view.paths.length == 0`? Should there be an extra`else if` in here?

Reply:
also, what about assigning `this.view.paths[this.stack.currentImageIdIndex]` to a var?